plugins/evm_passthrough/routes.py

```diff
@@ -56,7 +56,8 @@ def handle_request(evm, project_id, path=None):
         'PROJECT-ID': project_id,
         'API-TOKENS': str(g.project.api_token_count),
         'API-TOKENS-USED': str(g.project.used_api_tokens),
-        'API-TOKENS-REMAINING': str(g.project.api_token_count - g.project.used_api_tokens)
+        'API-TOKENS-REMAINING': str(g.project.api_token_count - g.project.used_api_tokens),
+        'EXPIRY-DATE':str(g.project.expires)
```

Comment:
Nice idea to return the expiration date in the header. I was actually planning to add the same feature. One small request, however:  Can we change this to be like this:
```python
'EXPIRATION-DATE':str(g.project.expires)
```
The reason is because there is already an "expiry_time" field returned by the `request_project` method, which represents the time when the prices quoted by the `request_project` method expire. To avoid confusion between the two kinds of expiry/expiration, I'm trying to keep their names different (expiry vs expiration) in the API docs.